frontend/src/action-creator.sync.js

```diff
@@ -197,16 +197,17 @@ export const APPLIED_FILTER = (searchType) => `AppliedFilter_${searchType}`
 export const setAppliedFilter = (key, value, searchType) => ({ type: `${SET}/${APPLIED_FILTER(searchType)}`, key, value })
 export const resetAppliedFilter = (searchType) => ({ type: `${RESET}/${APPLIED_FILTER(searchType)}` })
 
+export const EVERY_NOTIFICATIONS = 'EveryNotifications'
```

Comment:
I'd rather use the NOTIFICATION key and add "EVERY"

same for NOTIFICATION_LIST that should use NOTIFICATION